_rules/auto-update-text-efbfc7.md

```diff
@@ -0,0 +1,230 @@
+
+	<script type="text/javascript" src="/test-assets/efbfc7/script.js"></script>
+</body>
+```
+
+#### Passed Example 3
+
+The text content of the `span` element automatically updates multiple times without user intervention. A button is available to hide the automatically updating content. The rule is not applicable to the `p` element because it has a child (the `span` element) whose content updates.
+
+```html
+<body onload="startUpdates()">
+	<p>Random number: <span id="target">1</span></p>
+	<input type="button" onclick="hide()" value="Hide updating content" />
+
+	<script type="text/javascript" src="/test-assets/efbfc7/script.js"></script>
+</body>
+```
+
+#### Passed Example 4
+
+The text content of the `span` element automatically updates multiple times without user intervention. An [instrument][] is available to change the update frequency. The rule is not applicable to the `p` element because it has a child (the `span` element) whose content updates.
+
+```html
+<body onload="startUpdates()">
+	<p>Random number: <span id="target">1</span></p>
+	<label for="interval">Content update frequency (seconds):</label>
+	<input type="text" id="interval" />
+	<input type="button" onclick="changeFrequency(document.getElementById('interval').value)" value="Change frequency" />
+
+	<script type="text/javascript" src="/test-assets/efbfc7/script.js"></script>
+</body>
+```
+
+### Failed
+
+#### Failed Example 1
+
+The text content of the `span` element automatically updates multiple times without user intervention. There is no [instrument][] to stop, pause, hide or alter the frequency of the automatic updates.
+
+```html
+<body onload="startUpdates()">
+	<p>Random number: <span id="target">1</span></p>
+
+	<script type="text/javascript" src="/test-assets/efbfc7/script.js"></script>
+</body>
+```
+
+#### Failed Example 2
+
+The text content of the `span` element automatically updates multiple times without user intervention. A button is available to stop the automatic updates, but the button only stops the updates while it is focused.
+
+```html
+<body onload="startUpdates()">
+	<p>Random number: <span id="target">1</span></p>
+	<input type="button" onfocus="stopUpdates()" onblur="startUpdates()" value="Stop updates" />
+
+	<script type="text/javascript" src="/test-assets/efbfc7/script.js"></script>
+</body>
+```
+
+### Inapplicable
+
+#### Inapplicable Example 1
+
+The text content automatically updates but only as a result of the user activating a button on the page.
+
+```html
+<body>
+	<p>Random number: <span id="target">1</span></p>
+	<input type="button" id="control" onclick="startUpdates()" value="Start updates" />
+
+	<script type="text/javascript" src="/test-assets/efbfc7/script.js"></script>
+</body>
+```
+
+#### Inapplicable Example 2
+
+The automatically updating text content is the only content in the document.
+
+```html
+<body onload="startUpdates()">
+	<span id="target">1</span>
```

Comment:
While yes, this is correct, I'd also argue that all of the other examples are cases where this counter is the only content on the page. A button and a "random number" label would definitely be part of the auto-updating content. I think the examples need to be improved here; have some other content on them to make them valid. I also think an additional assumption is necessary for this. While yes, this "other text" solution works a little, this shows quite well that it's not an air tight test method.

Reply:
I did have this additional content in the examples previously and it got cut out by reviews, because technically the label was other text. 
I do agree that having additional content, unrelated to the updating content, makes the examples better, and I'll add it again.